src/common/scope.js

```diff
@@ -2,6 +2,20 @@ export const SCOPE_DATASET = 'dataset';
 export const SCOPE_GRAPHIC = 'graphic';
 export const SCOPE_COLLECTION = 'collection';
 export const SCOPE_DOCUMENT = 'document';
+
+export const sameScope = (scopeA, scopeB) => {
```

Comment:
This function return true in every case no :grin: ?

If a & b  = SCOPE_DOCUMENT
if a & b = SCOPE_COLLECTION
if a = SCOPE_COLLECTION & b = SCOPE_DOCUMENT 
if b = SCOPE_COLLECTION & b = SCOPE_DOCUMENT

Reply:
For these cases, yes
but when a = SCOPE_DATASET and b = SCOPE_GRAPHIC, then it's false

Reply:
OK, so can you use includes instead, and rename the function because it not only test for equality

Reply:
Use fantastic functional syntax from JulienD's big brain